examples/quad/main.rs

```diff
@@ -877,6 +878,9 @@ where
                 Some(&self.submission_complete_semaphores[frame_idx]),
             );
 
+            self.device
+                .wait_for_fence(&self.submission_complete_fences[frame_idx], !0)
```

Comment:
We can't do this. I mean, technically we can, but that would not be a good example if it waits on CPU for the GPU work to be done every frame...

Reply:
@kvark I introduced this to solve the following validation error. Is there a better way?

```
[2019-10-14T11:34:03Z ERROR gfx_backend_vulkan]
VALIDATION [VUID-vkDestroyFramebuffer-framebuffer-00892 (0)] : Cannot call vkDestroyFramebuffer on VkFramebuffer 0x59[] that is currently in use by a command buffer. The Vulkan spec states: All submitted commands that refer to framebuffer must have completed execution (https://www.khronos.org/registry/vulkan/specs/1.1-extensions/html/vkspec.html#VUID-vkDestroyFramebuffer-framebuffer-00892)
object info: (type: FRAMEBUFFER, hndl: 89)
```

Reply:
I'll need to look into this closer. The new swapchain API that we introduced recently is most difficult on Vulkan. It's not too easy to investigate.

Reply:
Actually, it's not our bug, see https://github.com/gfx-rs/gfx/issues/3015#issuecomment-531491026

Reply:
Okay, thank you for checking this. I'll remove the `wait_for_fence` call then.